src/meshapi/util/uisp_import/sync_handlers.py

```diff
@@ -241,6 +246,8 @@ def import_and_sync_uisp_links(uisp_links: List[UISPDataLink]) -> None:
                     message=f"Possible duplicate objects detected, links share the same UISP ID ({uisp_uuid})",
                 )
 
+            # XXX (wdn): Do we want to make a history record in this case? I _think_ so?
+            # maybe i dont understand this well enough
```

Comment:
Yeah I think we do want a record for this, since it's changing something we care about. It might be noisy, but that's UISP's fault :(